src/figuresClasses.ts

```diff
@@ -1,19 +1,71 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle {
+  public shape = 'triangle';
```

Comment:
use enum in other figures

```suggestion
  public shape = Shape.Triangle;
```